plugins/notifications-backend/src/service/router.ts

```diff
@@ -250,27 +258,41 @@ export async function createRouter(
     res.send(status);
   });
 
-  router.post('/update', async (req, res) => {
+  const updateNotifications = async (
+    user: string,
+    ids: string[],
+    read?: boolean,
+    saved?: boolean,
+  ) => {
+    if (read === true) {
+      await store.markRead({ user, ids });
+    } else if (read === false) {
+      await store.markUnread({ user, ids });
+    }
+
+    if (saved === true) {
+      await store.markSaved({ user: user, ids });
+    } else if (saved === false) {
+      await store.markUnsaved({ user: user, ids });
+    }
+  };
+
+  router.post('/notifications/update', async (req, res) => {
```

Comment:
In general, we should use just HTTP's GET, PUT, DELETE, we do not need `/update` in the URL.
The `PUT` seems to be a good fit here.

Reply:
@Rugvip had other thoughts of using PUT